docker/docker-compose/rcbe.yaml

```diff
@@ -0,0 +1,36 @@
+# This is a replace for run_docker.sh
+version: '2.3'
+services:
+  rcbe:
+    environment:
+      - OPER_HOME=/home/oper
+      - HUID
+      - HGID
+      - DISPLAY
+      - QT_X11_NO_MITSHM=1
+
+    image: "dev:${_container_tag}"
```

Comment:
Change this to `rcbe_dev`, re-tag image if needed.

Reply:
Done